### DISCUSSION TITLE
I'm hiring - Hyper Casual Games Studio in Berlin!

### DISCUSSION THREAD

**Discussion by sschmid** [sschmid/Entitas-CSharp#1012] [Announcements]
# Hi everyone!

I started a new mobile games company and we're building addicting hyper casual games with Unity for iOS and Android. I'm hiring talented devs, artists and game designers for a full-time position in our offices in Berlin!

### Where: Berlin!

Interested? PM me on the [Entitas Chat](https://gitter.im/sschmid/Entitas-CSharp) or reply here.

Not in Berlin? PM me anyways, we will find a solution :)

Looking forward to hearing from you

Happy coding :)
Simon

**Comment by VirtualMaestro** [comment]
Hey Simon!
My congratulations to you with such a decision!
You are implementing my dreams :)
Wish you all the best on this way!

P.S.: it's a pity you're not in Hamburg :)

**Comment by Josecarz** [comment]
Hi Simon, Im very interested in work with you, it is possible another way to contact ? Thanks man...

**Comment by LywOPUS** [comment]
Unfortunately, my English and German are not very good.

**Comment by AtheosCode** [comment]
cool

**Comment by nilayshah812** [comment]
Myself Nilay shah from India and I am interested in developing games. I know gimp, CPP language, also I am learning unity and csharp.

**Comment by Brainshack** [comment]
Hey Simon. If you are up for someone working part-time and remote from Cologne. I have my roots in web development (over ten years of devlopment experience), have done professional work in UE4  and also have done some Unity projects, though only pirvate stuff for game jams etc. However, I know how to write test-driven code and have used a bit of entitas aswell. If that sounds like something you might want to give a shot, let me know.
